decoders/connector/lansitec/nb-iot-solar-bluetooth-gateway/v1.0.0/payload.js

```diff
@@ -0,0 +1,315 @@
+// NB-IoT Bluetooth Gateway
```

Comment:
New decoders that are added must be made in TypeScript and must have a unit test.